examples/adjoints/adjoints_ex5/adjoints_ex5.C

```diff
@@ -173,16 +173,8 @@ void write_output(EquationSystems & es,
 #endif
 }
 
-void set_system_parameters(HeatSystem & system,
-                           FEMParameters & param)
+void set_system_parameters(FEMSystem &system, FEMParameters &param)
 {
-  // Use the prescribed FE type
-  system.fe_family() = param.fe_family[0];
-  system.fe_order() = param.fe_order[0];
-
-  // Use analytical jacobians?
-  system.analytic_jacobians() = param.analytic_jacobians;
-
```

Comment:
Okay, this whole commit needs a little reworking.  And get into the habit of looking at "git diff" before committing, and use "git add --patch" if it turns out that desired and undesired changes got mixed together some how?